contracts/src/v0.8/ccip/test/rmn/armProxy/ARMProxyStandaloneTestSetup.t.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity 0.8.24;
+
+import {ARMProxy} from "../../../rmn/ARMProxy.sol";
+import {Test} from "forge-std/Test.sol";
+
+contract ARMProxyStandaloneTestSetup is Test {
```

Comment:
Can probably rename this to just ARMProxySetup to be in like with all other setups